docs/reference/method/MongoDBCollection-countDocuments.txt

```diff
@@ -52,6 +52,14 @@ obtain the result. If a ``filter`` parameter is given, this is converted into
 a ``$match`` pipeline operator. Optional ``$skip`` and ``$limit`` stages are
 added between ``$match`` and ``group`` if present in the options.
 
+.. note::
+
+   This method counts documents on the server side. To obtain the total number
```

Comment:
I think it would be  clearer to say "To obtain an approximate total number of documents ...". When I read the current language, I first see  "the total number" which to me is the exact number, and when I read the rest of the sentence I mentally go back and revise "the total number" to become "an estimated number".

Reply:
Good point, updated!